ibis/backends/tests/test_export.py

```diff
@@ -44,6 +44,19 @@
 limit_no_limit = limit + no_limit
 
 
+def test_table___arrow_c_stream__(awards_players):
+    if not hasattr(pa.Table, "__arrow_c_stream__"):
+        pytest.skip("Arrow PyCapsule interface not available in this pyarrow version")
+    # Smoketest for __arrow_c_stream__ interface
+    # Here we just plumb through to the underlying pyarrow implementation
+    obj = awards_players.__arrow_c_stream__()
+    assert obj is not None
+
+    schema = awards_players.schema().to_pyarrow().__arrow_c_schema__()
+    obj2 = awards_players.__arrow_c_stream__(schema)
+    assert obj2 is not None
+
```

Comment:
You could add a check that `pa.table(awards_players)` runs (or check explicitly that it returns the same as `to_pyarrow`, like `pa.table(awards_players).equals(awards_players.to_pyarrow())`)

Reply:
Ah, that's a much better test, thanks!